k8s/upgrades/0.8.2-0.9.0/cstor/cstor-volume-update-082-090.yaml

```diff
@@ -259,6 +264,45 @@ spec:
     {{- verifyErr $verifyErrMessage true | saveAs "listTargetDeployment.verifyErr" .TaskResult | noop -}}
     {{- end }}
 ---
+apiVersion: openebs.io/v1alpha1
+kind: RunTask
+metadata:
+  name: upgrade-cstor-volume-082-090-pre-check-target-pod-phase
+  namespace: default
+spec:
+  meta: |
+    id: targetPodList
+    apiVersion: v1
+    kind: Pod
+    action: list
+    runNamespace: {{ .UpgradeItem.namespace }}
+    options: |-
+      labelSelector: openebs.io/persistent-volume={{ .UpgradeItem.name }},openebs.io/target=cstor-target
+  post: |
+    {{- $CustomJsonPath := printf "{.items[?(@.status.phase=='Running')].metadata.name}" -}}
+    {{- $ErrMsg := printf "target pod corresponding to volume: {%s} not in running phase" .UpgradeItem.name -}}
+
+    {{- jsonpath .JsonResult $CustomJsonPath | trim | saveAs "targetPodList.podName" .TaskResult | noop -}}
+    {{- .TaskResult.targetPodList.podName | notFoundErr $ErrMsg | saveIf "targetPodList.notFoundErr" .TaskResult | noop -}}
```

Comment:
verifyErr

Reply:
Updated the error message but notFoundErr is good. IMO we should go with verifyERR when we are making comparisons.